### ISSUE TITLE
Provide a way to exit CLI prompt

### ISSUE THREAD

**Issue by EvanClements** [tzlion/frogtool#5]
Usually Ctrl+c or Ctrl+x will allow me to get out of a command line prompt. There should be something in place to do the same here.

**Comment by tzlion** [comment]
This is on Linux I assume? Quitting with Ctrl-C works for me but maybe it's only working on Windows

**Comment by EvanClements** [comment]
Yeah, I'm using it on Linux Mint and Termux on Android and it didn't work when I tried it. I'm not saying it's an error, because it's honestly more of a preference than a need of any sort, and it quite possibly is just on my end.
